source/Ubuntu-RI.ufo/fontinfo.plist

```diff
@@ -95,9 +95,9 @@ The scope of the Ubuntu Font Family includes all the languages used by the vario
 		<array>
 		</array>
 		<key>openTypeNameUniqueID</key>
-		<string>Ubuntu Regular Version 0.83</string>
+		<string>Ubuntu Italic Version 1.000</string>
```

Comment:
Change in `Ubuntu-RI.ufo` from "Ubuntu Regular Version" to "Ubuntu Italic Version".  This is probably useful and correct, but should be a separate, clearly documented pull request, from the changing of the version number.

Reply:
Why not one "Set version metadata to 0.830 UFO Source" that fixes these and adds updated components to everything in one go?